.env

```
@@ -0,0 +1 @@
+GOPHERAI_API_KEY=rp4EP8mHxatUP4HlrkryoajwaZfvARf9GkYJdiaOdd9BI8PE
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)